### DISCUSSION TITLE
delploy the FundMe.sol get error.

### DISCUSSION THREAD

**Discussion by suddenly1990** [Cyfrin/foundry-full-course-cu#1974] [General]
when I learning Lesson4: 
I want to deploy the fundMe.sol,but I got this error:
<img width="553" alt="截屏2024-07-09 11 22 49" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/13513828/76392bee-e7b8-43f3-a56f-e698559a5818">
it is my code:
`
pragma solidity ^0.8.18;

interface AggregatorV3Interface {
  function decimals() external view returns (uint8);

  function description() external view returns (string memory);

  function version() external view returns (uint256);

  function getRoundData(
    uint80 _roundId
  ) external view returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound);

  function latestRoundData()
    external
    view
    returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound);
}

contract FundMe {
    function fund() public  payable  {

        require(msg.value > 1e18, "didn't send enought");

    }

    function getPrice() public {
    

    }

    function getConversionRate() public {}

    function getVersion() public view returns(uint256) {
        return AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306).version();

    }
}
`
I want to know why I can't use the metamusk test network

**Comment by suddenly1990** [comment]
I know Disconnect metamusk and reconnected the remix

**Comment by EngrPips** [comment]
What error did you get, @suddenly1990?
